### DISCUSSION TITLE
API or command to automatically manage file in Documents for our RAG purpose.

### DISCUSSION THREAD

**Discussion by piaopiao20** [open-webui/open-webui#3500] [General]
we need a API or command to manage(delete/update) file in Document every night for our RAG purpose.
here is our case:
1. we use spider to  crawl web site every night , and the result will be stored in server
2. once craw complete, we plan to use script to upload file to Open web UI.(delete old and upload new one)

we found some API printed in log for Document but not sure whether we can invoke it from out side.

so please consider this feature.
or 
any workaround or suggestion?

thanks

**Comment by loviuz** [comment]
Hi @piaopiao20 ,

I don't know how to update the file, but you can delete it in this way:

```bash
curl -X DELETE $HOST'/api/v1/files/<your-file-id>' --header 'Authorization: Bearer '$API_KEY

```
